dspace-spring-rest/src/main/java/org/dspace/app/rest/IdentifierRestController.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ * <p>
```

Comment:
Currently, Travis CI is throwing a "Missing License header" error. I think it's because your license header has a `<p>` tag (likely inserted by your IDE), while the version compared against has a blank line, see: https://github.com/DSpace/DSpace/blob/master/LICENSE_HEADER

If you are using IntelliJ, you might want to change your IDE settings to no longer insert a `<p>` on blank Javadocs lines.  See last bullet (staring "Finally, update IDEA's Javadoc settings to not insert "<p>"") at: https://wiki.duraspace.org/display/DSPACE/Code+Style+Guide#CodeStyleGuide-IntelliJIDEA

Reply:
You can also double check this change locally via maven by running:
`mvn license:check`  (which will verify all classes have the appropriate license header)

Reply:
Wow, IntelliJ doesn't seem to want to reformat comments after the fact.  I had all the right boxes ticked, but I still had to remove this manually.  Thanks for the mvn tip!!